iam-drift-detection/detect_drift.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+ORGANIZATION_ID=$1
+BUCKET_QUERY="labels:terraform"
```

Comment:
Is there any way to make this label value configurable? Can be a follow up

Reply:
Done.